qa/tasks/cephfs/test_admin.py

```diff
@@ -970,6 +972,58 @@ def test_single_path_rootsquash(self):
             cmdargs = ['echo', 'some random data', Raw('|'), 'sudo', 'tee', filepath]
             self.mount_a.negtestcmd(args=cmdargs, retval=1, errmsg='permission denied')
 
+    def test_multifs_single_path_rootsquash(self):
```

Comment:
QA run was unsuccessful - https://pulpito.ceph.com/xiubli-2024-08-05_05:14:48-fs-wip-xiubli-testing-20240802.055215-squid-distro-default-smithi/7836646/

```
2024-08-05T08:52:05.179 INFO:tasks.cephfs_test_runner:======================================================================
2024-08-05T08:52:05.179 INFO:tasks.cephfs_test_runner:ERROR: test_multifs_single_path_rootsquash (tasks.cephfs.test_admin.TestFsAuthorize)
2024-08-05T08:52:05.179 INFO:tasks.cephfs_test_runner:Test root_squash with multi fs
2024-08-05T08:52:05.179 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2024-08-05T08:52:05.179 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2024-08-05T08:52:05.179 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_229250be86d9b9262624e6bf4f333bb0720b3db3/qa/tasks/cephfs/test_admin.py", line 1851, in tearDown
2024-08-05T08:52:05.179 INFO:tasks.cephfs_test_runner:    self.mount_a.umount_wait()
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_229250be86d9b9262624e6bf4f333bb0720b3db3/qa/tasks/cephfs/kernel_mount.py", line 191, in umount_wait
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:    self.umount(force)
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_229250be86d9b9262624e6bf4f333bb0720b3db3/qa/tasks/cephfs/kernel_mount.py", line 170, in umount
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:    raise e
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_229250be86d9b9262624e6bf4f333bb0720b3db3/qa/tasks/cephfs/kernel_mount.py", line 163, in umount
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:    self.client_remote.run(args=cmd, timeout=UMOUNT_TIMEOUT, omit_sudo=False)
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_teuthology_4bc527c722abc094dbc33e04519a3647224076f6/teuthology/orchestra/remote.py", line 523, in run
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:    r = self._runner(client=self.ssh, name=self.shortname, **kwargs)
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_teuthology_4bc527c722abc094dbc33e04519a3647224076f6/teuthology/orchestra/run.py", line 455, in run
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:    r.wait()
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_teuthology_4bc527c722abc094dbc33e04519a3647224076f6/teuthology/orchestra/run.py", line 161, in wait
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:    self._raise_for_status()
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_teuthology_4bc527c722abc094dbc33e04519a3647224076f6/teuthology/orchestra/run.py", line 181, in _raise_for_status
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:    raise CommandFailedError(
2024-08-05T08:52:05.180 INFO:tasks.cephfs_test_runner:teuthology.exceptions.CommandFailedError: Command failed on smithi031 with status 32: 'sudo umount /home/ubuntu/cephtest/mnt.0'
```

Reply:
@rishabh-d-dave The QA run was unsuccessful but you approved the change at the same time?